daisy_workflows/image_build/windows/README.md

```diff
@@ -0,0 +1,19 @@
+### Windows ISO to Google Compute Image Build
```

Comment:
Can you call out that this is just 2016

Reply:
I added a table detailing the supported WS versions and recommended updates to include.